Buildings/Fluid/HeatExchangers/DXCoils/AirCooled/Examples/DXCooledDataCenter.mo

```diff
@@ -0,0 +1,297 @@
+within Buildings.Fluid.HeatExchangers.DXCoils.AirCooled.Examples;
+model DXCooledDataCenter
+  "Example that illustrates the use of Buildings.Fluid.HeatExchanger.DXCoil in a data center room"
+  extends Modelica.Icons.Example;
+  replaceable package Medium =
+      Buildings.Media.Air;
+
+  // Air temperatures at design conditions
+  parameter Modelica.SIunits.Temperature TASup_nominal = 273.15+18
+    "Nominal air temperature supplied to room";
+  parameter Modelica.SIunits.Temperature TRooSet = 273.15+24
+    "Nominal room air temperature";
+  parameter Modelica.SIunits.Temperature SATSet = 273.15+17.78
+    "Nominal room air temperature";
+ /////////////////////////////////////////////////////////
+  // Cooling loads and air mass flow rates
+  parameter Modelica.SIunits.HeatFlowRate QRooInt_flow=
+     1000 "Internal heat gains of the room";
+  parameter Modelica.SIunits.HeatFlowRate QRooC_flow_nominal=
+    -2*QRooInt_flow;
+  parameter Modelica.SIunits.MassFlowRate mA_flow_nominal=
+    1.3*QRooC_flow_nominal/1006/(TASup_nominal-TRooSet)
+    "Nominal air mass flow rate, increased by factor 1.3";
+  parameter Modelica.SIunits.TemperatureDifference dTFan = 2
+    "Estimated temperature raise across fan that needs to be made up by the cooling coil";
+  parameter Modelica.SIunits.HeatFlowRate QCoiC_flow_nominal=
+    QRooC_flow_nominal
+    "Cooling load of coil";
+
+  Buildings.Examples.ChillerPlant.BaseClasses.SimplifiedRoom datCenRoo(
+    nPorts=2,
+    redeclare package Medium = Medium,
+    rooLen=50,
+    rooHei=3,
+    rooWid=40,
+    QRoo_flow=QRooInt_flow,
+    m_flow_nominal=mA_flow_nominal)
+    "Simplified data center room"
+    annotation (Placement(transformation(extent={{38,32},{58,12}})));
+  Buildings.Fluid.HeatExchangers.DXCoils.AirCooled.VariableSpeed varSpeDX(
+    redeclare package Medium = Medium,
+    dp_nominal=400,
+    datCoi=datCoi,
+    minSpeRat=0,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    T_start=303.15)
+    "Variable speed DX coil"
+    annotation (Placement(transformation(extent={{26,-76},{46,-56}})));
+  Buildings.BoundaryConditions.WeatherData.ReaderTMY3 weaDat(
+    pAtmSou=Buildings.BoundaryConditions.Types.DataSource.Parameter,
+    filNam="/home/yangyangfu/Bitbucket/modelica-buildings/Buildings/Resources/weatherdata/USA_MA_Boston-Logan.725090_TMY2.mos")
+    "Weather data reader"
```

Comment:
@YangyangFu When setting the weather file location to be like that, I wonder if it will cause problem when running the test in Travis? Would it be better to put the file in location like `"modelica://Buildings/Resources/weatherdata/USA_MA_Boston-Logan.725090_TMY2.mos"`